src/cascadia/WpfTerminalControl/TerminalTheme.cs

```diff
@@ -58,32 +58,32 @@ public struct TerminalTheme
         /// <summary>
         /// The default background color of the terminal, represented in Win32 COLORREF format.
         /// </summary>
-        public uint DefaultBackground;
+        public readonly uint DefaultBackground;

```

Comment:
This changes the public API contract of `TerminalTheme`, doesn't it? We don't know how our consumers are using it.

Reply:
Addressed!

Reply:
I'm sorry, I'm not totally certain about the fix here. I saw one commit that removed `readonly` from a DispatcherTimer, but in the final PR the fields in `TerminalTheme` (which is part of our public API) are still marked `readonly`.